my-app/components/navbar/navbar.module.scss

```diff
@@ -0,0 +1,29 @@
+.main{
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 4.5rem;
+  padding: 0 0.5rem;
+  .navLinks{
+    ul{
+      display: flex;
+      align-items: center;
+      li{
+        list-style: none;
+        margin-right: 2.5rem;
```

Comment:
We'll use the short hand notation for the margin and padding in the whole project so kindly replace it with short hand notation